python/jupytergis_core/package.json

```diff
@@ -91,6 +91,7 @@
     "access": "public"
   },
   "jupyterlab": {
+    "schemaDir": "schema",
```

Comment:
Line 17 of this file, you want to list the schema folder as part of the package file